cmd/juju/commands/synctools_test.go

```diff
@@ -203,7 +203,7 @@ func (s *syncToolsSuite) TestSyncToolsCommandDeprecatedDestination(c *gc.C) {
 	messages := []jc.SimpleMessage{
 		{loggo.INFO, "Use of the --destination flag is deprecated in 1.18. Please use --local-dir instead."},
 	}
-	// Run sync-tools command with --destination flag.
+	// Run sync-agents command with --destination flag.
 	ctx, err := s.runSyncToolsCommand(c, "-m", "test-target", "--destination", dir, "--stream", "released")
```

Comment:
can we fix the helper function while we're here?

Reply:
done